components/Loadingspinner.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { ActivityIndicator, StyleSheet, Text, View } from 'react-native'
+
+const Loadingspinner = () => (
+	<View style={[styles.container, styles.horizontal]}>
+		<ActivityIndicator size='large' color='#eabd91' />
+	</View>
+)
+
+const styles = StyleSheet.create({
+	container: {
+		flex: 1,
+		justifyContent: 'center',
+	},
+	horizontal: {
+		flexDirection: 'row',
+		justifyContent: 'space-around',
+		padding: 10,
+	},
+})
```

Comment:
I'm a bit confused, the loading spinner is round and centered on desktop but on mobile a bit to the left and has another styling...